pdcupdater/consumer.py

```diff
@@ -51,9 +51,15 @@ def __init__(self, hub):
         super(PDCUpdater, self).__init__(hub)
 
     def consume(self, msg):
-        msg = msg['body']  # Remove envelope
+        # Remove the envelope
+        headers = msg.get('headers', {})  # https://github.com/mokshaproject/moksha/pull/35
+        topic, msg = msg['topic'], msg['body']
```

Comment:
Can you use a different variable name here instead of `msg`? Overriding the the parameter is confusing.